api/MediaRecorder.json

```
@@ -228,9 +228,7 @@
           "spec_url": "https://w3c.github.io/mediacapture-record/#dom-mediarecorder-onerror",
           "support": {
             "chrome": {
-              "version_added": "49",
-              "partial_implementation": true,
-              "notes": "The interface for this event is a plain <a href='https://developer.mozilla.org/docs/Web/API/Event'><code>Event</code></a>, not <a href='https://developer.mozilla.org/docs/Web/API/MediaRecorderErrorEvent'><code>MediaRecorderErrorEvent</code></a>. See <a href='https://crbug.com/1250432'>bug 1250432</a>."
+              "version_added": "49"
```

The bug is still open right? I think it still needs to become an ErrorEvent instead of a plain Event?

```suggestion
               "version_added": "49",
              "partial_implementation": true,
              "notes": "The interface for this event is a plain <a href='https://developer.mozilla.org/docs/Web/API/Event'><code>Event</code></a>, not <a href='https://developer.mozilla.org/docs/Web/API/ErrorEvent'><code>ErrorEvent</code></a>. See <a href='https://crbug.com/1250432'>bug 1250432</a>."
```